bootstrap.sh

```diff
@@ -320,12 +320,11 @@ add_dependency OPC_ENABLED "mbedtls"
 
 USE_SHARED_LIBS=${TRUE}
 TESTS_DISABLED=${FALSE}
+ASAN_ENABLED=${FALSE}
 
 ## name, default, values
 add_multi_option BUILD_PROFILE "RelWithDebInfo" "RelWithDebInfo" "Debug" "MinSizeRel" "Release"
 
-add_disabled_option ASAN_ENABLED ${FALSE} "ASAN_BUILD"
```

Comment:
Are you sure this could be removed? 
For eg., settings are persisted without this?

Reply:
nice, yes indeed, it has to be explicitly added to `save_state`

Reply:
Indeed, it was missing from `bt_state`.  Fixed in https://github.com/apache/nifi-minifi-cpp/pull/1005/commits/28970606a01063e4491bbb35ca4de5899e1cd865.

This bootstrap script is a bit of a mess, and should be refactored, but I think that should be done in a separate PR.

Reply:
Agreed.